### PR TITLE
LPD-27917 - Management Toolbar Clear results button does not show cursor pointer

### PR DESCRIPTION
**Pull request by juanjofgliferay**
## Task
[LPD-27917](https://liferay.atlassian.net/browse/LPD-27917) / [LPP-54403](https://liferay.atlassian.net/browse/LPP-54403)

## Issue
Clear results button does not show the cursor pointer when hovering over it. It is caused because we are using a link element without `href` attribute. 
It seems to be a regression issue, as it started having an href attribute that was changed to use a callback handler.

## Solution
Replace the link by a button, to trigger the handler and that displays the cursor of type pointer by default.

image

### CODE REVIEW

**Comment by juanjofgliferay** [comment]
ci:test:playwright

File: modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/management_toolbars.jsp [hunk 1]
```diff
@@ -15,53 +15,61 @@ ClaySampleManagementToolbarsDisplayContext managementToolbarsDisplayContext = ne
...
+		searchInputAutoFocus="<%= true %>"
+		searchInputName="mySearchInputName"
+		selectable="<%= true %>"
+		sortingOrder="desc"
+		viewTypeItems="<%= managementToolbarsDisplayContext.getViewTypeItems() %>"
+	/>
+</section>
 
-<h3>ACTIVE STATE</h3>
+<section id="management_toolbar_active_state">
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/management_toolbars.jsp:35]: A convention in DXP is for `id` to be `camelCase`.

**Review by markocikos — COMMENTED** [review]
@juanjofgliferay just a minor SF nit, otherwise looks good!

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! :sparkles:

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
[PW test report](https://testray-old.liferay.com/reports/production/logs/2024-06/test-1-36/test-portal-acceptance-pullrequest%28master%29/6242/playwright-js-tomcat90-mysql57-jdk8/6/0/playwright-report/index.html#?testId=0dabb0a80581e203e384-e2f31f42971bbd114d6d) :tada: 
image

**Comment by juanjofgliferay** [comment]
ci:forward

**Pull request closed without merging**